# launch.py->node->name
gimbal_demo_node:
  ros__parameters:
    pid_yaw_remote_in: [8000.0, 0.0, 0.0] 
    pid_yaw_remote_out: [10.0, 0.0, 0.0]  
    pid_yaw_init_in: [1.8, 0.0, 0.0]  
    pid_yaw_init_out: [2.0, 0.0, 0.0]  
    pid_pitch_in: [8000.0, 0.0, 0.0]
    pid_pitch_out: [10.0, 0.0, 0.0]
    pid_ammor: [1.0, 0.0, 0.0]
    pid_ammol: [1.0, 0.0, 0.0]
    pid_rotor: [3.0, 0.0, 0.1]
chassis_demo_node:
  ros__parameters:
    pid1: [1.8, 0.00, 0.3]
    pid2: [1.8, 0.00, 0.3]
    pid3: [1.8, 0.00, 0.3]
    pid4: [1.8, 0.00, 0.3]  
    pid_follow: [1.5, 0.000, 0.3]

    
